sphinx/ext/autodoc/__init__.py

```diff
@@ -1585,6 +1589,10 @@ def add_directive_header(self, sig: str) -> None:
                 self.add_line('   ' + _('Bases: %s') % ', '.join(bases), sourcename)
 
     def get_object_members(self, want_all: bool) -> Tuple[bool, ObjectMembers]:
+        def convert(m: ClassAttribute) -> ObjectMember:
+            """Convert ClassAttribute object to ObjectMember."""
+            return ObjectMember(m.name, m.value, class_=m.class_, docstring=m.docstring)
```

Comment:
meaningless conversion. `get_class_members()` should return `ObjectMember` instead of `ClassAttribute`...
 I'll do it next step.